modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,139 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.5
```

Comment:
Per our docs guidelines each heading needs its own ID. I notice that few if any of the headings in this module have an ID. You don't have to add IDs for headings that are already merged in this module, but for each heading that you are adding in this PR, please add a unique ID:
```suggestion
[id="new-features-ossm-2-5"]
== New features {SMProductName} version 2.5
```

It's a pain, but without these IDs added manually to ensure uniqueness, the platform autogenerates them based on the text in the heading. If you ever have duplicate headings within an assembly/module, this creates duplicate IDs and I imagine that causes issues for our site.